test/docs/operation_test.rb

```diff
@@ -39,8 +38,64 @@ def create_model(ctx, params:, **)
       result.success?.must_equal true
       result[:model].text.must_equal "Enjoy an IPA"
     end
+
+    it "allows indifferent access for ctx keys" do
+      #:ctx-indifferent-access
+      result = Memo::Create.(params: { text: "Enjoy an IPA" })
+
+      result[:params]     # => { text: "Enjoy an IPA" }
+      result['params']    # => { text: "Enjoy an IPA" }
+      #:ctx-indifferent-access end
+
+      result.success?.must_equal true
+      result[:params].must_equal({ text: "Enjoy an IPA" })
+      result['params'].must_equal({ text: "Enjoy an IPA" })
+    end
+
+    it "allows defining aliases for ctx keys" do
+      module AliasesExample
+        Memo = Struct.new(:text)
+
+        module Memo::Contract
+          Create = Struct.new(:sync)
+        end
+
+        #:ctx-aliases-step
+        class Memo::Create < Trailblazer::Operation
+          #~flow
+          step ->(ctx, **) { ctx[:'contract.default'] = Memo::Contract::Create.new }
+          #~flow end
+
+          pass :sync
+
+          def sync(ctx, contract:, **)
+            # ctx['contract.default'] == ctx[:contract]
+            contract.sync
+          end
+        end
+        #:ctx-aliases-step end
+      end
+
+      #:ctx-aliases
+      options = { params: { text: "Enjoy an IPA" } }
+      flow_options = {
+        context_options: {
+          aliases: { 'contract.default': :contract, 'policy.default': :policy },
```

Comment:
:ballot_box_with_check:  We really really have to note in the docs that the keys are SYMBOLS... I had it a bunch of times now that I used `"contract.default" => :contract` and it would say `missing keyword: contract`. Can you maybe add that to http://trailblazer.to/2.1/docs/activity.html#activity-troubleshooting, too? :heart:

Reply:
Yeah agree, will do!

Reply:
Let me think about this more. 
But how about we store symbol version of key in the alias store too ? So setting any of them would make it accessible for string + symbol.

Reply:
We can leave the behavior as it is, but bold-print a note in the docs. That should be fine...

Reply:
Yeah agree, I found it sometimes adds unnecessary time computation.